### DISCUSSION TITLE
Evidential thresholds?

### DISCUSSION THREAD

**Discussion by KLABDB3K** [PygmalionOfCyprus/cmo-db-requests#1371] [Q&A] [answered]
What is the threshold of evidence of putting new weapons and associated loadouts into the database given how problematic it is to remove erroneous additions? 
From my experience of researching the IRIS-T SLM following the announcement they were being sent to Ukraine, the initial impression was that this was in service with Germany but on further digging and checks its clearly apparent its not, and potentially Luftwaffe service was never on the cards. 
So there are numerous brochure weapons which are advertised but dont ever get procured for operational use. In particular Rosboronexport has a catalogue of weaponry thats not actually operational or its in service date is no where near as advertised.
The laser guided S-25 S-8 etc may be in a brochure and well documented from numerous books and websites but theres negligible evidence of operational deployment with the RuAF and the photos there are show markings which are clearly test rounds, dayglo orange patches etc.
I'm thinking of examples such as the Adder R-77 / R-77-1 saga etc, the Vikhr saga with No significant order for either system until the 2013/2015 era for weapons designed in the mid to late 90's.
By example the standard Su-25SM2/3 AT-16 Vikhr loadouts were bogus and  were eventually removed but not before some scenarios had the aircraft and loadout embedded. 
I think it would be useful to set down what does constitute evidence?

**Comment by PygmalionOfCyprus** [comment, marked as answer]
I should note that the IRIS-T SLM, while not in service with Germany or Ukraine (yet), _is_ in service with Egypt; it was on those grounds that we added it in 495.

But I think this question draws a bit of a false equivalency between hypothetical/unfielded weapons and "erroneous additions." 

In my book, an "erroneous addition" is something truly and egregiously _wrong_. For example, accidentally creating a variant of a missile that doesn't exist, never was supposed to exist, and will never exist. (The examples of this in the DB I've found usually happened with Russian/Chinese equipment that was mislabeled by observers and then discovered years later to be something else entirely.) These are, as you mentioned, both "bogus" and also very hard to deal with without potentially screwing up scenarios that have been made using them. We try to avoid these for obvious reasons.

On the other hand, a hypothetical addition could be anything from an interesting weapon concept (e.g. the proposed Iowa conversions), an extant weapon with no buyers (laser-guided S-25s, etc.), or a rumored addition to a country's inventory (IRIS-T SLM for Ukraine). They exist in some modellable form, and someone might want to experiment with their effectiveness or create a "what if" scenario with them. I don't think there's anything wrong with adding these to the DB, and in some cases I believe they're just as important to make available as "real" platforms. The only limiting factor is our time: given a choice between spending time to add a missing "real" platform and a hypothetical, we're usually going to choose the former.

**Comment by KLABDB3K** [comment]
Thank you for taking the time to answer.
I can see the logic behind it now which is very useful. 
I think the key may be to aim for a future marker of not just hypoyhetical platforms but hypothetical loadouts where the scenario designer can enable or disable the option in the design stages.
Thanks again.
